### PR TITLE
remove PATH/.profile stuff from post-install message

### PR DESCRIPTION
**Pull request by air**

### CODE REVIEW

File: dcos-cli/install-dcos-cli-ea2.sh [hunk 1]
```diff
@@ -60,7 +60,6 @@ dcos config set package.sources '["https://github.com/mesosphere/universe/archiv
 dcos package update
 
 echo "Finished installing and configuring DCOS CLI."
-echo "Please add $VIRTUAL_ENV_PATH/bin to your PATH."
-echo "On Linux systems, run the line below to automatically set up your PATH: "
-echo "echo \"source $ENV_SETUP\" >> ~/.profile"
-echo "Once your PATH is set up, type dcos help to get started."
+echo "On Linux systems, run the line below to set up the DCOS environment:"
```
Comment (jsancio) [inline dcos-cli/install-dcos-cli-ea2.sh]: Linux and MacOS X. Technically anything that has bash/sh.

Comment (air) [inline dcos-cli/install-dcos-cli-ea2.sh]: Maybe the whole statement is bizarre, since we're running in a bash script?

**Comment by jsancio** [comment]
Let's also change the file `dcos-cli/install-dcos-cli.sh`

**Comment by jsancio** [comment]
LGTM! :shipit:

**Pull request merged**